packages/web/components/ui/media-audio-node.tsx

```
@@ -17,25 +16,25 @@ export const AudioElement = withHOC(
     return (
-      <PlateElement {...props} className="mb-1">
+      <PlateElement {...props} className='mb-1'>
         <figure
-          className="group relative cursor-default"
+          className='group relative cursor-default'
           contentEditable={false}
         >
-          <div className="h-16">
-            <audio className="size-full" src={unsafeUrl} controls />
+          <div className='h-16'>
+            <audio className='size-full' src={unsafeUrl} controls />
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)